tests/test_enterprise.yaml

```diff
@@ -35,3 +35,37 @@
     - url: '/enterprise/v1/catalogs'
     - method: 'GET'
     - expected_status: [400]
+
+- test:
+    - name: 'Enterprise catalog endpoint returns HTTP 200'
+    - url: '/enterprise/v1/enterprise-catalogs'
+    - method: 'GET'
+    - headers: {'Authorization': 'aeiou'}
+    - expected_status: [200]
+
+- test:
+    - name: 'Enterprise catalog by UUID endpoint returns HTTP 200'
+    - url: '/enterprise/v1/enterprise-catalogs/0123456789abcdefg'
+    - method: 'GET'
+    - headers: {'Authorization': 'aeiou'}
+    - expected_status: [200]
+
+- test:
+    - name: 'Program by UUID endpoint returns HTTP 200'
+    - url: '/enterprise/v1/programs/0123456789abcdefg'
+    - method: 'GET'
+    - headers: {'Authorization': 'aeiou'}
+    - expected_status: [200]
+
+- test:
+    - name: 'Enterprise catalog POST is disabled'
+    - url: '/enterprise/v1/enterprise-catalogs'
+    - method: 'POST'
+    - headers: {'Authorization': 'aeiou'}
+    - expected_status: [405]
+
+- test:
+    - name: 'Enterprise catalog GET rejected without authorization'
+    - url: '/enterprise/v1/enterprise-catalogs'
+    - method: 'GET'
+    - expected_status: [400]
```

Comment:
Maybe just being a little picky, but can we add tests to check for GET/POST requests being rejected without authorization for all endpoints? That might blow up the size of the file a little so let me know if it's unnecessary. In the same spirit, can we check that POST is disabled for `/enterprise-catalogs/{uuid}` as well as programs?